src/main/scala/com/goyeau/kubernetes/client/api/PodsApi.scala

```diff
@@ -48,7 +48,8 @@ private[client] case class NamespacedPodsApi[F[_]](
   val F: Async[F],
   val resourceEncoder: Encoder[Pod],
   val resourceDecoder: Decoder[Pod],
-  val listDecoder: Decoder[PodList]
+  val listDecoder: Decoder[PodList],
+  val contextShift: ContextShift[IO]
```

Comment:
```suggestion
  contextShift: ContextShift[F]
```
With this we are not forcing the IO monad

Reply:
No need anymore. Changed to Async